amcl/test/basic_localization_stage.xml

```diff
@@ -44,6 +46,6 @@ setting pose: 30.329 34.644 3.142
       <param name="initial_pose_y" value="21.421"/>
       <param name="initial_pose_a" value="-1.003"/>
     </node>
-  <test time-limit="180" test-name="basic_localization_stage" pkg="amcl" 
-        type="basic_localization.py" args="0 47.060 21.603 -1.053 0.75 0.75 90.0"/>
+    <test time-limit="180" test-name="basic_localization_stage" pkg="amcl"
+          type="basic_localization.py" args="0 30.329 34.644 3.142 0.75 0.75 90.0"/>
```

Comment:
Is there a way to make this less 'magic number-y'. Perhaps explicitly naming these as x, y, z and theta. Because right now it is not clear what these numbers mean

Reply:
That would extend the argument handling a lot. So keeping it this way has my preference.